### PR TITLE
Updated roadmap and added planned features of version 1.2.

### PR DESCRIPTION
**Pull request by netopyr**

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -455,6 +455,32 @@ More pre-releases are planned during the next months.
...
+<tr class="feature-missing">
+    <td>Logs cleanup</td>
+    <td>1</td>
+    <td>
+        The name and structure of the log-files need to be cleaned up. 
+    </td>
+</tr>
+
+</table>
+
```
Comment (hendrikebbers) [inline features.md:483]: Table in markdown can be done without HTML:
```
{:.table}
| Header1      | Header2 	         | Header 2         | 
| ------------| -------------- | -------------- |
| value 1         | value 2            | value 3            |
| ROW 2         | value 4            | value 5            |
```

Comment (hendrikebbers) [reply features.md:483]: https://www.tablesgenerator.com/markdown_tables

Comment (netopyr) [reply features.md:483]: But can you also style them the way we do?

Comment (hendrikebbers) [reply features.md:483]: ah, we have custom styles per row. I do not know if that is working. you can add CSS classes for the table but not for a specific row as far as I know. Anyway, I do not have a problem with a HTML table here. Markdown table is better readable in the source but if not all features are supported I would go with HTML

**Pull request merged**